os/binfmt/libelf/libelf_bind.c

```diff
@@ -170,7 +200,9 @@ static int elf_relocate(FAR struct elf_loadinfo_s *loadinfo, int relidx, FAR con
 	int i;
 
 	/* Read the relocation table into memory */
-	elf_readreltab(loadinfo, relsec);
+	if (elf_readreltab(loadinfo, relsec) == -ENOMEM) {
+		elf_enable_caching(loadinfo);
```

Comment:
Enabling cache cannot be guaranteed, I think.
How it will be if fail to initialize the cache?

Reply:
Okay. Thanks.